ui/pages/onboarding-flow/recovery-phrase/review-recovery-phrase.js

```diff
@@ -3,16 +3,16 @@ import { useHistory, useLocation } from 'react-router-dom';
 import PropTypes from 'prop-types';
 import Box from '../../../components/ui/box';
```

Comment:
We could remove this deprecated `Box` and import the component-library verion. The props should all work as we are using enums now
```suggestion
```

Reply:
Resolved